provisioner/clusterpy.go

```diff
@@ -1137,6 +1146,38 @@ func performDeletion(ctx context.Context, logger *log.Entry, client dynamic.Inte
 	return nil
 }
 
+func overrideDeletionProtection(ctx context.Context, kind, name string, iface dynamic.ResourceInterface) error {
+	annotation := ""
+	switch kind {
+	case "Namespace":
+		annotation = "zalando.org/delete-namespace"
+	case "Postgresql":
+		annotation = "zalando.org/delete-clustername"
+	default:
+		// no annotation needed
+		return nil
+	}
+
+	patch := map[string]interface{}{
+		"metadata": map[string]interface{}{
+			"annotations": map[string]interface{}{
+				"zalando.org/delete-date": time.Now().Format("2006-01-02"),
+				annotation:                name,
```

Comment:
`name` isn't correct for Postresql, isn't it? Because the annotation name is `"zalando.org/delete-clustername"` the value seems to be the cluster's name and not the database itself.

I also wouldn't mind leaving out Postgresql entirely here. We don't need it right now and this way we cannot accidentally delete it.

Reply:
> We don't need it right now and this way we cannot accidentally delete it.

I agree, I will remove it then